packages/web-components/fast-element/src/__test__/helpers.ts

```diff
@@ -11,6 +11,6 @@ export function toHTML(node: Node, preserveCommentMarkup = false): string {
 }
 
 let counter = 0;
-export function uniqueElementName() {
+export function uniqueElementName(): string {
```

Comment:
Does this mean we can't infer *any* types?

Reply:
No, this is just fixing an eslint error.